packages/matrix/tests/commands.spec.ts

```diff
@@ -212,7 +221,8 @@ test.describe('Commands', () => {
     expect(boxelMessageData.context.tools).toMatchObject([]);
   });
 
-  test(`it does not include patch tool in message event when top-most card is read-only`, async ({
+  // TODO: currently we need isolated realm server to get payment setup to work
```

Comment:
This test will fail as the base url get 404, currently we are only can get /user on isolated realm server (4205)

GET application/vnd.api+json http://localhost:4201/_user: 404